docs/DeveloperGuide.md

```diff
@@ -182,11 +182,14 @@ The `AddCommand` class is responsible for adding new patient's information in th
 #### Specifications
 
 * AddCommand, as defined by the `AddCommand` class, contain parameters which consists of: 
-  *  `patientHospitalId` integer, 
-  * `name`, `preferredName` String with only alphabets character,
-  *  `foodPreference`, `familyCondition`, `hobby` and `tag` which are alphanumeric.
+  *  `patientHospitalId` non-negative numeric characters which uniquely identifies the patient, 
+  * `name`, `preferredName` String which contains alphanumeric characters,
```

Comment:
does it **only** contain alphanumeric characters? or can it contain alphanumeric characters and others?

Reply:
To be exact, alphanumeric characters and spaces. Will do further explanation on the doc.